libnmstate/nm/route.py

```diff
@@ -141,3 +149,40 @@ def _get_default_route_config(gateway, metric, default_table_id, iface_name):
         Route.NEXT_HOP_ADDRESS: gateway,
         Route.METRIC: metric,
     }
+
+
+def add_routes(setting_ip, routes):
+    default_gateway_defined = False
+    for route in routes:
+        if route[Route.DESTINATION] in (IPV4_DEFAULT_GATEWAY_DESTINATION,
+                                        IPV6_DEFAULT_GATEWAY_DESTINATION):
+            if default_gateway_defined:
+                raise NmstateNotImplementedError(
+                    'Only a single default gateway is supported.')
+            default_gateway_defined = True
+            _add_route_gateway(setting_ip, route)
+        else:
+            _add_specfic_route(setting_ip, route)
+
+
+def _add_route_gateway(setting_ip, route):
+    setting_ip.props.gateway = route[Route.NEXT_HOP_ADDRESS]
+    if route[Route.TABLE_ID] != NM_ROUTE_TABLE_USE_DEFAULT_CFG:
+        setting_ip.props.route_table = route[Route.TABLE_ID]
+    if route[Route.METRIC] != NM_ROUTE_DEFAULT_METRIC:
+        setting_ip.props.route_metric = route[Route.METRIC]
+
+
+def _add_specfic_route(setting_ip, route):
+    destination, prefix_len = route[Route.DESTINATION].split('/')
+    prefix_len = int(prefix_len)
+    family = socket.AF_INET6 if iplib.is_ipv6_address(destination) \
+        else socket.AF_INET
+    metric = route.get(Route.METRIC, NM_ROUTE_DEFAULT_METRIC)
+    next_hop = route[Route.NEXT_HOP_ADDRESS]
+    rt = nmclient.NM.IPRoute.new(family, destination, prefix_len, next_hop,
```

Comment:
IMHO it is better to not abbreviate the variable here and use `route`.

Reply:
Changed to `ip_route` for `NM.IPRoute`.